Assets/Scripts/State/IdleState.cs

```diff
@@ -26,11 +29,41 @@ public IdleState(Character character, State nextState = null)
 
         public override void Update(float deltaTime)
         {
+                Tile startTile = character.CurrTile;
```

Comment:
Is there any use of assigning CurrTile to another variable that's never changed?